### PR TITLE
Bugfix/no more clickdeb

### PR DESCRIPTION
**Pull request by mvo5**
This branch kills the compatibility with the clickdeb system. 

There are currently some tests skipped that need to be ported, I would appreciate helper from e.g. @chipaca with this if he has time to look at this. Plus all the click* stuff needs to be renamed.

### CODE REVIEW

File: snap/file.go [hunk 1]
```diff
@@ -27,18 +27,15 @@ import (
 
 // File is the interface to interact with the low-level snap files
 type File interface {
-	Verify(allowUnauthenticated bool) error
-	Close() error
-	UnpackWithDropPrivs(targetDir, rootDir string) error
```
Comment (niemeyer) [inline snap/file.go]: Woo... lovely!

File: snap/file.go [hunk 2]
```diff
@@ -27,18 +27,15 @@ import (
...
+	// Install installs the file
```
Comment (niemeyer) [inline snap/file.go]: I'd drop the comment.. (Yeah, Snap.Install installs the snap). :)

**Comment by niemeyer** [comment]
LGTM!

**Comment by niemeyer** [comment]
The DEBIAN/manifest issue got fixed as well, so I suppose the revert needs to be reverted? :)

**Comment by mvo5** [comment]
@niemeyer I reverted this compatibility again  in 97bcb01898977a8940903d534513de13a5bb5e15

**Comment by chipaca** [comment]
:+1:

**Pull request merged**